src/components/settings/PushNotifications/hooks/useNotificationRegistrations.ts

```diff
@@ -19,7 +20,7 @@ const registrationFlow = async (registrationFn: Promise<unknown>, callback: () =
 
     // Gateway will return 200 with an empty payload if the device was (un-)registered successfully
     // @see https://github.com/safe-global/safe-client-gateway-nest/blob/27b6b3846b4ecbf938cdf5d0595ca464c10e556b/src/routes/notifications/notifications.service.ts#L29
-    success = response == null
+    success = isEmpty(response)
```

Comment:
Why do we need this check? If we get to this point then we've gotten a 200, no?

Reply:
I also think so otherwise if it threw we would end up in the catch. I assume this was added in case the response is successful but not empty. For now I would leave it as is since the notifications are being reworked on the service side so we will have to touch this again anyway.